aospy/__init__.py

```diff
@@ -1,6 +1,9 @@
 """aospy: management, analysis, and plotting of gridded climate data."""
 from .__config__ import (user_path, LAT_STR, LON_STR, PFULL_STR, PHALF_STR,
```

Comment:
I think we should consider having a module or dictionary with all the _STR variables.  That way the import could be a single object.  I.e. if it was a module called `internal_names` then we could do `from . import internal_names` and then `internal_names.LAT_STR` etc.  That would clean up import statements in a lot of places.